mlir/lib/Bindings/Python/IRTypes.cpp

```diff
@@ -742,7 +736,6 @@ class PyFunctionType : public PyConcreteType<PyFunctionType> {
         "inputs",
         [](PyFunctionType &self) {
           MlirType t = self;
-          auto contextRef = self.getContext();
```

Comment:
Why this change? Being able to utilize multiple context and use the one one is in, is quite nice (I'll check more below still)

Reply:
That's dead code. I can make it make do what you're describing but it currently doesn't.